testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2623,6 +2626,54 @@ public void testInsertInTransaction()
         }
     }
 
+    @Test
+    public void testSelectAfterInsertInTransaction()
```

Comment:
nit (Feels like diminishing returns to test every single operation): Might want to test DDL as well?

Reply:
it's really Hive-only, and we want to remove it (AFAIK), so i wouldn't spend much time on this.